.prettierrc

```
@@ -0,0 +1,5 @@
+{
+    "trailingComma": "all",
+    "tabWidth": 4,
```

Usually, it is better to use:

```suggestion
"tabWidth": 2,
```